bot/exts/advent_of_code/_cog.py

```diff
@@ -107,8 +107,8 @@ async def completionist_task(self) -> None:
         placement_leaderboard = json.loads(leaderboard["placement_leaderboard"])
 
         for member_aoc_info in placement_leaderboard.values():
-            if member_aoc_info["stars"] != 50:
-                # Only give the role to people who have completed all 50 stars
+            if member_aoc_info["stars"] <= _helpers.STARS_THIS_YEAR:
```

Comment:
```suggestion
            if member_aoc_info["stars"] < _helpers.STARS_THIS_YEAR:
```
It would continue even if user has the same amount of stars as `STARS_THIS_YEAR`